birdhouse/components/stac/config/proxy/conf.extra-service.d/stac.conf

```diff
@@ -0,0 +1,26 @@
+
+    location /stac {
+        # We need the first `/stac` for service resolution.
+        # We need the second `/stac` for API redirect in STAC (see `root-path` and `ROUTER_PREFIX`).
+        # See https://github.com/stac-utils/stac-fastapi/issues/427
+        # See https://github.com/crim-ca/stac-app/blob/main/stac_app.py#L60
+        proxy_pass https://10.0.0.9/twitcher/ows/proxy/stac/stac;
+        proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Host $host:$server_port;
+        proxy_buffering off;
+        include /etc/nginx/conf.d/cors.include;
+    }
+
+    location /stac-browser/ {
+        proxy_pass http://stac-browser:8080/;
```

Comment:
Missing a note about what we decided for protecting the browser as discussed (protection through STAC API sufficient).